src/probnum/__init__.py

```diff
@@ -24,17 +24,11 @@
 """
 # pylint: disable=wrong-import-order
 
-from . import diffeq
-from . import filtsmooth
-from . import linalg
-from . import quad
-from . import random_variables
-from . import utils
-
-from .random_variables import asrandvar, RandomVariable
-
 # -*- coding: utf-8 -*-
-from pkg_resources import get_distribution, DistributionNotFound
+from pkg_resources import DistributionNotFound, get_distribution
```

Comment:
@JonathanWenger what do you think about the changes to this file? I don't really have an opinion here, but the structure looks quite a bit different. I also just saw that we explicitly disable pylint here. Should we do the same for isort, or does it look fine the way it was formatted?

All other changes in this PR did not seem controversial to me, but if you have the time it could also be beneficial if you could skim through the changes to double-check.

Reply:
@nathanaelbosch Does anyone know why pylint was disabled here? We can make isort ignore this as well (comments, flags, etc.)

Reply:
The changes are fine! Pylint was disabled since package imports were (unnecessarily) before third party imports. You can simply remove `# pylint: disable=wrong-import-order` with the changes you made. Following that we can merge this from my side.